packages/augur-core/source/contracts/MKRShutdownHandler.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity 0.5.10;
+
+import 'ROOT/ICash.sol';
+import 'ROOT/external/IDaiVat.sol';
+import 'ROOT/libraries/math/SafeMathUint256.sol';
+
+
+contract MKRShutdownHandler {
+    using SafeMathUint256 for uint256;
+
+    ICash public cash;
+    IDaiVat public vat;
+
+    uint constant DAI_ONE = 10 ** 27;
```

Comment:
VatDAI is `10**45`
DAI is `10**18`

Neither is `10**27`.

Reply:
The constant in their codebase is called `ONE`. As that doesn't really make sense in the context of Augur I wasn't sure on the best name. Perhaps `MKR_ONE`??

Reply:
It doesn't make sense in the context of their codebase either.  😉

Ignoring gas costs, my recommendation would be to add a couple methods for `vatDaiToDai` and `daiToVatDai` and inside those just do `value.mul(10**27)` and `value.div(10**27)` directly (no constant).  The goal would be to isolate the Maker weirdness into well named functions and not try to figure out how to name that weird scalar multiplier.

Reply:
We are already referencing and dealing with MKR code/contracts and making it a magic value seems wrong. that said the function idea does sound more readable so I'll rename the const and add those funcs

Reply:
Perhaps call it `RAY` if you want to go with Maker naming scheme?

Reply:
"ONE" is the new name in their contracts, but i agree it is terrible. The name and the value differ by `10^27-1`.